Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1465,7 +1482,7 @@ def identityFS(wf):
 from  Configuration.PyReleaseValidation.upgradeWorkflowComponents import *
 
 defaultDataSets={}
-defaultDataSets['2017']='CMSSW_8_1_0_pre1-80X_upgrade2017_design_v4_UPG17-v'
+defaultDataSets['2017']='CMSSW_8_1_0_pre2-80X_upgrade2017_realistic_v1_UPG17realGT-v'
```

Comment:
Hi Hengne,

the request was there for more than a week.
It looks like it has fallen through the cracks.
Maybe an email to dataops to get it going?